src/main/java/FindCommand.java

```diff
@@ -1,17 +1,19 @@
-import java.util.ArrayList;
-
 public class FindCommand extends Command {
 
-    String command;
-    
-    public FindCommand(String command){
-        this.command = command;
+    public FindCommand(String command, TaskList taskList ){
+        super(command, taskList);
     }
 
-    public void execute(TaskList taskList){
+    @Override
+    public String processCommand(){
+        return super.command;
+    }
+
+    @Override
+    public String execute(String processedCommand){
         TaskList taskListsWithKeyWords = new TaskList();
         
-        int length = taskList.size();
+        int length = super.taskList.size();
 
         for(int i = 0; i < length; i++){
```

Comment:
Perhaps it might be possible to implement the <Iterable> interface for the TaskList object to make the iteration across all tasks easier.